media-libs/phonon/phonon-9999.ebuild

```diff
@@ -44,10 +44,10 @@ RDEPEND="
 		designer? ( dev-qt/designer:5 )
 	)
 	pulseaudio? (
-		dev-libs/glib:2
-		>=media-sound/pulseaudio-0.9.21[glib]
+		dev-libs/glib:2[${MULTILIB_USEDEP}]
+		>=media-sound/pulseaudio-0.9.21[glib,${MULTILIB_USEDEP}]
 	)
-	zeitgeist? ( dev-libs/libqzeitgeist )
+	zeitgeist? ( dev-libs/libqzeitgeist[${MULTILIB_USEDEP}] )
```

Comment:
I'd say it's better to just remove `[${MULTILIB_USEDEP}]` and enable zeitgeist only for the native ABI until the dep gets converted. This is what I did e.g. for the rarely used backends of `dev-qt/qtsql:4`.